### DISCUSSION TITLE
Why the model I trained is latest.pdparams rather than best_accuracy.pdparams

### DISCUSSION THREAD

**Discussion by Haeppypuppy** [PaddlePaddle/PaddleOCR#12364] [Q&A]
Please provide the following information to quickly locate the problem

when I used the pre_train model to train. However,the name of output model in C:\Users\PaddleOCR\output 
is latest.pdparam rather than best_accuracy.pdparams

did I get any wrong step?

System Environment：Windows 
Paddle：  release 2.7
PaddlePaddle: 2.3.1

Command Code：python tools/train.py -c configs/det/det_mv3_db.yml -o Global.pretrained_model=./pretrain_models/MobileNetV3_small_x0_35_pretrained
Complete Error Message：NA

We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): Yes

Please try to not include the image in the issue.

**Comment by cuicheng01** [comment]
1. Please make sure you have actually used the eval data set
2. Please make sure you have actually performed the eval operation

**Comment by Haeppypuppy** [comment]
I found the setting and problem.

In  program.py

`
save_model(
    model,
    optimizer,
    save_model_dir,
    logger,
    config,
    is_best=True,  # 
    prefix='best_accuracy',  # 將prefix設置為'best_accuracy'
    best_model_dict=best_model_dict,
    epoch=epoch,
    global_step=global_step)
    `
is_best=True,  # True ==save the best model when training
is_best=False,  # False ==save the latest model when training
